src/service/viewer-impl.js

```diff
@@ -608,6 +608,29 @@ export class Viewer {
   }
 
   /**
+   * Triggers "tick" event for the viewer.
+   * @param {!JSONObject} message
+   */
+  tick(message) {
+    this.sendMessage_('tick', message, false);
+  }
+
+  /**
+   * Triggers "sendCsi" event for the viewer.
+   */
+  sendCsi() {
+    this.sendMessage_('sendCsi', undefined, false);
```

Comment:
This name should probably be changed to something more generic, but I think it makes sense to stick with this for now.